internal/testdata/config.toml

```diff
@@ -32,5 +32,5 @@
 	PreParamsGenerationTimeout = "6m37s"
 	PreParamsTargetPoolSize = 36
 
-[Extensions.TBTC]
+[ethereum.ContractAddresses.TBTC]
```

Comment:
> in case we add another application

If we add another application, it'll still be an Ethereum contract if it's on Ethereum, and a different one if it's not. I'm not sure I follow, but the current structure didn't make a lot of sense to me---definitely open to revising though, I didn't get a chance to dig into the PRs and read the reasoning on this.

Reply:
I think the current structure makes sense in case the application has more configurable properties than just Ethereum contract address or when it has more than one contract address to configure. I am not sure where to, for example, put BTC configuration properties for liquidated deposit recovery. This is something TBTC-specific, so we could have all those properties grouped together under `[Extensions.TBTC]`.

Reply:
The recovery stuff is a great point. I think it may make sense to consider that chain-centrically as well though---we haven't historically had a `bitcoin` namespace in the config, but that may make sense here...

On the other hand I wonder if an operator may prefer to use some recovery data for one chain and different recovery data for a different one… I could, for example, see them using different xpubs with different derivation paths in these cases to make it easier to reconcile accounting for liquidations. So I don't think we _can_ consider these the same thing. I think we should nest the configurations under the host chain, and consider providing a way to set “defaults” for those who don't want to configure per-chain.

Reply:
I like the idea about `bitcoin` namespace. My concern is how much the configuration properties we need to add are BTC-specific or TBTC-specific. For example, `ElectrsURL` should nicely fit into `bitcoin` namespace. `MaxFeePerVByte` should fit nicely into both `bitcoin` and `extensions.tbtc`. On the other hand, `BTCRefunds` fits better into `extensions.tbtc` and clearly describes what this property is for.

Reply:
> I think we should nest the configurations under the host chain, and consider providing a way to set “defaults” for those who don't want to configure per-chain.

This is where I'm settling here, fwiw.